backbone/src/main/res/values/strings.xml

```diff
@@ -32,74 +31,30 @@
     <string name="rsb_consent_section_more_info_withdrawing">Learn more about withdrawing</string>
 
     <!-- Consent sharing step -->
```

Comment:
Please make sure you remove comments that are not needed anymore. This applies to several comments here